src/App.tsx

```diff
@@ -12,155 +17,26 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
```

Comment:
Add the Navbar with 2 links Home and People. Active one should have the has-background-grey-lighter class;
![image](https://github.com/mate-academy/react_people-table-basics/assets/63243136/1f7a6c1f-065a-4dff-8c17-f10fbef5d3d8)